js/popup.js

```diff
@@ -0,0 +1,56 @@
+const GUESTS_WORD_FORMS = [
+  'гостя',
+  'гостей',
+  'гостей',
+];
+const ROOMS_WORD_FORMS = [
+  'комната',
+  'комнаты',
+  'комнат',
+];
+const TYPES = {
+  palace: 'Дворец',
+  flat: 'Квартира',
+  house: 'Дом',
+  bungalow: 'Бунгало',
+};
+
+const popupList = document.querySelector('.map__canvas');
+const popupTemplate = document.querySelector('#card').content.querySelector('.popup');
+const popup = popupTemplate.cloneNode(true);
+
+const photosContainer = popup.querySelector('.popup__photos');
+const featuresContainer = popup.querySelector('.popup__features');
+
+const getWordForm = function (num, wordForms) {
+  if (num > 1 && num < 5) return wordForms[1];
+  if (num === 1) return wordForms[0];
```

Comment:
Должны быть обязательные блоки кода
```suggestion
  if (num > 1 && num < 5) {
    return wordForms[1];
  }
  if (num === 1) {
    return wordForms[0];
  }
```